packages/helpers/classes/attachment.js

```diff
@@ -39,17 +40,29 @@ class Attachment {
     data = toCamelCase(data);
 
     //Extract properties from data
-    const {content, filename, type, disposition, contentId} = data;
+    const {content, filename, type, disposition, contentId, file} = data;
+
+    let fileContent = !file ? content : this.readFile(file);
 
     //Set data
-    this.setContent(content);
+    this.setContent(fileContent);
     this.setFilename(filename);
     this.setType(type);
     this.setDisposition(disposition);
     this.setContentId(contentId);
   }
 
   /**
+   * Read a file and return its content as base64
+   */
+  readFile(file) {
+    let encoding = !file.encoding ? 'utf8' : file.encoding;
+    let data = fs.readFileSync(file.path);
+    let buff = Buffer.from(data, encoding);
```

Comment:
All `let`'s are `const`ant and I'm wondering if we should return the buffer, instead of converting to string at this point? That way you can use the helper function to read files and get buffers and manipulate them if needed before passing to the attachment.

Also, we can update the Attachment class to detect if a Buffer is passed and automatically convert to string.

Reply:
Thanks for the tip.
Should I convert the method readFile to a helper function? Do you think that another helper function to convert a string to Buffer is necessary? 

According to [fs.readFileSync documentation](https://nodejs.org/api/fs.html#fs_fs_readfilesync_path_options) if the `encoding` is passed, then a string is returned, otherwise it'll return a buffer. 

Perhaps in this case the buffer conversion using an encoding is completely unnecessary, just if the user want to get a string from the buffer, then the encoding would be useful.

Reply:
Might not need to expose it, as it's basically just using `readFileSync`. Let's pass the encoding straight to `readFileSync`, so we won't need to convert to buffer.